src/actions/types.js

```diff
@@ -6,8 +6,6 @@ export const LOGIN_USER_ERROR = 'LOGIN_USER_ERROR';
 export const SOCIAL_LOGIN_INITIATED = 'SOCIAL_LOGIN_INITIATED';
 export const SOCIAL_LOGIN_SUCCESS = 'SOCIAL_LOGIN_SUCCESS';
 export const GET_ALL_ARTICLES_SUCCESS = 'GET_ALL_ARTICLES_SUCCESS';
-
-// Profile action types
 export const GET_PROFILE_PAYLOAD = 'GET_PROFILE_PAYLOAD';
-export const GET_PROFILE_ERROR = 'GET_PROFILE_ERROR';
 export const GET_PROFILE_INITIATED = 'GET_PROFILE_INITIATED';
+export const LOGOUT_USER = 'LOGOUT_USER';
```

Comment:
Since this action deals with a `profile retrieval error` can it be named accordingly? Just a suggestion.

Reply:
@mendozabree , the action type `LOGOUT_USER` is being used to logout a user if their token has expired. It will also be used to logout a user when the logout button is pressed. The naming is therefore appropriate.

Reply:
That's fine then!